torch/csrc/jit/codegen/cuda/fusion.h

```diff
@@ -237,6 +237,18 @@ class TORCH_CUDA_CU_API Fusion : public IrContainer {
     return io_alias_;
   }
 
+  //! Interface to disable nvfuser zero
+  //!  in the generated kernel.
+  void disableNvFuserZero() {
```

Comment:
Is this actually necessary?

Also, if yes, can it be defined in `Kernel`? That seems like a better place to define it.

Reply:
In this PR, no, this is not needed. But in the future, the matmul scheduler will disable nvfuser using this interface. I don't think schedulers has access to kernel, so it has to be in fusion.

Reply:
How about making it an option to the `GpuLower` constructor?

We pass some compile- and run-time related properties schedulers down to the executor without associating them directly with a particular fusion. It seems to me that using nvfuser_zero or not is also such a property.

Reply:
Currently, the only such property is `index_type` I think? In fusion executor, it is easy to detect the size of the tensor and determine the index type, but it is hard to let fusion executor to detect "this is a matmul, so disable nvfuser zero".

Reply:
There's also the optional max register count set by the normalization scheduler. In certain cases, the scheduler may choose to set the max register count that is passed to the nvrtc compilation.

Reply:
OK, changed